Orange/canvas/registry/qt.py

```diff
@@ -296,7 +296,8 @@ def tooltip_helper(desc):
     tooltip = []
     tooltip.append("<b>{name}</b>".format(name=escape(desc.name)))
 
-    if desc.project_name and desc.project_name != "Orange":
+    if desc.project_name and (not desc.project_name.startswith('Orange') or
```

Comment:
We could also remove the additional condition and just show (from package_name) for all projects. This might come in handy if (when) we decide to separate canvas and core widgets.